opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "1.2"
+maintainer: "dave.scott@eu.citrix.com"
+authors: [ "Dave Scott" ]
+homepage: "https://github.com/xapi-project/ocaml-qmp"
+bug-reports: "https://github.com/xapi-project/ocaml-qmp/issues"
+tags: [
+  "org:mirage"
+  "org:xapi-project"
+]
+build: [
+  ["./configure" "--prefix" prefix]
+  [make]
+]
+build-test: [
+    ["./configure" "--prefix" prefix "--enable-tests"]
+    [make "test"]
+]
+install: [make "install" "BINDIR=%{bin}%"]
+remove: [
+  [make "uninstall" "BINDIR=%{bin}%"]
+  ["ocamlfind" "remove" "qmp"]
+]
+depends: [
```

Comment:
Should we also add the `base-unix` package, as the `unix` ocamlfind package is a dependency in the `_oasis` file?